stage2/README.md

```diff
@@ -49,10 +49,9 @@
 
 ### 06/06/2023
 
-- [Project on migration of JS project to TS](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/migration-newip-to-ts.md)
-- [Module: Typescript: Advanced](modules/typescript-advanced/)
+- [Module: Code review]()
+- [Module: Spa]()
```

Comment:
Imho, _Module: **SPA**_ would look better, now it looks like a resort 💆 🛀

Reply:
This is a feature; you don't know what will be in this module